core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -327,6 +327,10 @@ public function __set($strKey, $varValue)
 					case 'robots':
 						$htmlHeadBag->setMetaRobots($varValue);
 						break;
+
+					case 'noSearch':
+						$htmlHeadBag->setMetaRobots($varValue ? 'index,nofollow' : 'noindex,nofollow');
```

Comment:
Makes no sense to differ between `follow` and `nofollow` here. Imho if something is dynamically set to `noindex` (which I consider to be not a use case anymore - the core doesn't need it itself), it's irrelevant whether it was set to be followed or not.

Reply:
If `noSearch === true` it sets `index/nofollow`? And `noindex...` if `noSearch === false`?

Reply:
Yeah that's wrong of course.

Reply:
Fixed.